wcfsetup/install/files/lib/data/language/LanguageEditor.class.php

```diff
@@ -1054,5 +1056,10 @@ public static function copyLanguageContent($sourceLanguageID, $destinationLangua
         }
 
         PageEditor::resetCache();
+
+        EventHandler::getInstance()->fire(new CopyLanguageContent(
+            LanguageFactory::getInstance()->getLanguage($sourceLanguageID),
+            LanguageFactory::getInstance()->getLanguage($destinationLanguageID)
+        ));
```

Comment:
```suggestion
        EventHandler::getInstance()->fire(new CopyLanguageContent(
            LanguageFactory::getInstance()->getLanguage($sourceLanguageID),
            LanguageFactory::getInstance()->getLanguage($destinationLanguageID)
        ));
```

This appears to throw an exception in `LanguageAddForm`, likely due to some caching issue?

> wcf\system\language\event\CopyLanguageContent::__construct(): Argument #2 ($targetLanguage) must be of type wcf\data\language\Language, null given, called in /var/www/html/lib/data/language/LanguageEditor.class.php on line 1047

The simple solution would be using:

```suggestion
        EventHandler::getInstance()->fire(new CopyLanguageContent(
            new Language($sourceLanguageID),
            new Language($destinationLanguageID)
        ));
```